### PR TITLE
Docs: Add missing key to enable SigV4 for provisioning Elasticsearch data source

### LINKED ISSUE #29439: SigV4 Elasticsearch external datasource not reachable and not properly initialized
**What happened**: I am using a yaml datasource for Elasticsearch hosted on AWS with SigV4. Each time Grafana starts, my datasource is ingested. But I cannot use it or fetch any data through it. In the options panel, the parameter "SigV4 auth" is disabled.

Like this: https://ibb.co/n7S2c4t

**What you expected to happen**: I except the auth option "SigV4 auth" to be enabled when SigV4 credentials are defined.

What I expect is: https://ibb.co/7R3v8dG

**How to reproduce it (as minimally and precisely as possible)**:

1. Create an AWS Elasticsearch

2. In AWS, create an IAM user and associated role

3. Create an external datasource file with this configuration and complete with your AWS creds:

```yaml
apiVersion: 1

deleteDatasources:
  - name: ELK (eu-central-1)
    orgId: 1

datasources:
- name: ELK (eu-central-1)
  type: elasticsearch
  access: proxy
  database: "[logs.default.]YYYY.MM.DD"
  url: https://vpc-xxx.eu-central-1.es.amazonaws.com
  jsonData:
    esVersion: 70
    timeField: "@timestamp"
    interval: Daily
    logMessageField: message
    logLevelField: fields.level
    sigV4AuthType: keys
    sigV4AssumeRoleArn: arn:aws:iam::xxx:role/xxx-grafana
    sigV4Region: eu-central-1
  secureJsonData:
    sigV4AccessKey: xxx
    sigV4SecretKey: xxx
  orgId: 1
  editable: false
```

4. Start Grafana and see the datasource configuration, the option "SigV4 auth" is disabled

**Other things we need to know:**

- I use an Amazon Elasticsearch behind a private VPC endpoint

**Environment**:
- Grafana version:v7.3.3 (2489dc4d3a)
- Data source type & version: Elasticsearch bundled with Grafana
- OS Grafana is installed on: RedHat UBI 8.2
- User OS & Browser: Firefox, MacOS
- Grafana plugins: petrslavotinek-carpetplot-panel v0.1.1, grafana-clock-panel v1.1.1, jdbranham-diagram-panel v1.7.1, grafana-piechart-panel v1.6.1, natel-plotly-panel v0.0.6, vonage-status-panel v1.0.10, flant-statusmap-panel v0.3.4, 
- Others: Elasticsearch 7.8, grafana-worldmap-panel v0.3.2

### PR DESCRIPTION
**Pull request by wbrowne**
**Which issue(s) this PR fixes**:

Fixes #29439

**Special notes for your reviewer**:
Turn `hide whitespace changes` on to ignore the extra whitespace fixes

### CODE REVIEW

**Review by ivanahuckova — APPROVED** [review]
LGTM 🚀

**Review by aknuds1 — APPROVED** [review]

**Comment by aknuds1** [comment]
@wbrowne Are you sure docs should be backported to v7.3.5? Aren't they generated from master??

File: docs/sources/administration/provisioning.md [hunk 1]
```diff
@@ -147,19 +147,20 @@ Since not all datasources have the same configuration settings we only have the
...
 | tlsSkipVerify           | boolean | _All_                                                            | Controls whether a client verifies the server's certificate chain and host name.            |
 | graphiteVersion         | string  | Graphite                                                         | Graphite version                                                                            |
-| timeInterval            | string  | Prometheus, Elasticsearch, InfluxDB, MySQL, PostgreSQL and MSSQL  | Lowest interval/step value that should be used for this data source                         |
-| httpMode                | string  | Influxdb                                                          | HTTP Method. 'GET', 'POST', defaults to GET                                                |
-| httpMethod              | string  | Prometheus                                                       | HTTP Method. 'GET', 'POST', defaults to GET                                                |
+| timeInterval            | string  | Prometheus, Elasticsearch, InfluxDB, MySQL, PostgreSQL and MSSQL | Lowest interval/step value that should be used for this data source                         |
```
Comment (oddlittlebird) [inline docs/sources/administration/provisioning.md:150]:
```suggestion
| timeInterval            | string  | Prometheus, Elasticsearch, InfluxDB, MySQL, PostgreSQL and MSSQL | Lowest interval/step value that should be used for this data source.                         |
```

**Comment by oddlittlebird** [comment]
> @wbrowne Are you sure docs should be backported to v7.3.5? Aren't they generated from master??

Correct. Docs are generated from master.

**Comment by wbrowne** [comment]
@aknuds1 wasn't totally sure if that's the case for all docs so I figured no harm - but will remove as it's unnecessary. Thanks for clarifying @oddlittlebird!

**Pull request merged**